pubnub-gson/pubnub-gson-impl/src/main/java/com/pubnub/internal/java/endpoints/objects_api/members/GetChannelMembersImpl.java

```diff
@@ -19,19 +21,22 @@
 import java.util.Collection;
 import java.util.Collections;
 
+import static com.pubnub.internal.java.endpoints.objects_api.members.SetChannelMembersImpl.getMemberInclude;
+
 @Setter
 @Accessors(chain = true, fluent = true)
-public class GetChannelMembersImpl extends DelegatingEndpoint<PNMemberArrayResult, PNGetChannelMembersResult> implements GetChannelMembers {
+public class GetChannelMembersImpl extends DelegatingEndpoint<PNMemberArrayResult, PNGetChannelMembersResult> implements GetChannelMembers, GetChannelMembersBuilder {
 
     private final String channel;
     private Integer limit = null;
     private PNPage page;
     private String filter;
     private Collection<PNSortKey> sort = Collections.emptyList();
-    private boolean includeTotalCount;
-    private boolean includeCustom;
-    private boolean includeType;
-    private Include.PNUUIDDetailsLevel includeUUID;
+    private boolean includeTotalCount; // deprecated
```

Comment:
This is the case:
```
pubNubUnderTest.setChannelMembers()
                .channel(testChannelId)
                .uuids(channelMembers)
                .includeTotalCount(true)
                .includeCustom(true)
                .includeUUID(UUID_WITH_CUSTOM)
                .sync();
```
There is deprecation on setChannelMembers() so I thought this would be enough. I didn't want to pollute code by adding e.g.:

```
    /**
     * @deprecated Use {@link #include} instead.
     */
    @Deprecated
    private boolean includeTotalCount;
```
I can add this. What do you think?